core/src/main/java/io/knotx/server/handler/http/response/writer/ServerResponse.java

```diff
@@ -81,7 +82,7 @@ private MultiMap getHeaders(Set<String> allowedResponseHeaders) {
       clientResponse.getHeaders()
           .names()
           .stream()
-          .filter(header -> headerFilter(allowedResponseHeaders, header))
```

Comment:
It looks that we can safely remove `headerFilter` method.

Reply:
Done